guides/common/modules/con_how-to-use-iss-based-on-your-network-setup.adoc

```diff
@@ -0,0 +1,53 @@
+= How to Use ISS Based on Your Network Setup
+
+There are different ways of using ISS to synchronize content between two {ProjectServer}s.
+The way you can use depends on your multi-server setup that can fall to one of the following scenarios.
+
+.ISS Network Sync in a Disconnected Scenario
+In a disconnected scenario, there is the following setup:
+
+* The upstream {ProjectServer} is connected to the Internet.
+This server synchronizes content from the Red Hat Content Delivery Network (CDN) or custom sources.
+* The downstream {ProjectServer} is completely isolated from all external networks.
+* The downstream {ProjectServer} can communicate with a connected upstream {ProjectServer} over an internal network.
+
+ifndef::satellite[]
+image::disconnected_non_airgapped.png[title="The {Project} ISS Disconnected Scenario", alt="The {Project} ISS Disconnected Scenario"]
+endif::[]
+ifdef::satellite[]
+image::sync_servers_disconnected_sat.png[title="The {Project} ISS Disconnected Scenario", alt="The {Project} ISS Disconnected Scenario"]
+endif::[]
+
+You can synchronize CDN and custom content from the upstream {ProjectServer} to the downstream {ProjectServer} over the network. See xref:synchronizing-content-over-a-network_{context}[].
+
+.ISS Export Sync in an Air-Gapped Scenario
+In an air-gapped scenario, there is the following setup:
+
+* The upstream {ProjectServer} is connected to the Internet.
+This server synchronizes content from the Red Hat CDN or custom sources.
+* The downstream {ProjectServer} is completely isolated from all external networks.
+* The downstream {ProjectServer} does not have a network connection to a connected upstream {ProjectServer}.
+
+ifndef::satellite[]
+image::disconnected_airgapped.png[title="The {Project} ISS Air-Gapped Scenario", alt="The {Project} ISS Air-Gapped Scenario"]
+endif::[]
+ifdef::satellite[]
+image::sync_servers_airgapped_sat.png[title="The {Project} ISS Air-Gapped Scenario", alt="The {Project} ISS Air-Gapped Scenario"]
+endif::[]
+
+The only way for an air-gapped downstream {ProjectServer} to receive content updates is by exporting payload from the upstream {ProjectServer}, bringing it physically to the downstream {ProjectServer} and importing the payload.
+
+You have two options how to synchronize your content using the export sync:
+
+. Using the Upstream {ProjectServer} as a Content Store
++
+The upstream {ProjectServer} is mainly used as a content store for updates.
+The downstream {ProjectServer} serves as the main {ProjectServer} for managing content for all infrastructure behind the isolated network.
+You manage Content Views and Lifecycle Environments on the downstream {ProjectServer}.
+
+. Using the Upstream {ProjectServer} for Managing Content View Versions
++
+The upstream {ProjectServer} is not only used as a content store, but also is used to manage content.
+Updates coming from the CDN are curated into Content Views and Lifecycle Environments.
```

Comment:
Either `a CDN` or differentiated as above.

Reply:
Maybe without an article?
@melcorr Can you weight in as a native speaker, pls?

Reply:
@jeremylenz Or you? :) ^^